src/fetch_satellite_singlepass.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import pandas as pd
```

Comment:
these two imports are unused (likely shows as greyed out in your IDE) remove them for a marginal performance improvement and cleaner code :)